### DISCUSSION TITLE
Best way to support the project (donations etc.)

### DISCUSSION THREAD

**Discussion by StefanVDWeide** [jellyfin/Swiftfin#1064] [Questions] [answered]
Hi,

I've been using Swiftfin for Apple TV for half a year now and so far it has been absolutely great! I have the occasional issue with audio tracks and subtitles, but nothing that hasn't already been addressed for the upcoming release.

I see how tirelessly @LePips is handeling all the incoming issues and "next release when" questions and as a dev myself I know how ungrateful this can feel. 

Now, I was wondering if there is a good way to support the project, through donations for example, in order to better show our gratitude. I'm opening this discussion to get some more input from other user, contributes and @LePips himself!

**Comment by LePips** [comment, marked as answer]
Thank you for your interest in supporting the project and me personally!

The Jellyfin project has an OpenCollective page that is used for all projects, which we use for equipment and such.
- https://opencollective.com/jellyfin

To support me personally, I set up my GitHub sponsors only a little bit ago. To be clear, I do not feel like contributions are necessary nor will I be taking donations for certain features (bounties). However, I am grateful that you have thought contributing to me personally.

**Reply by Alarez777** [reply]
Hi, maybe it doesn't have much to do with the subject but I wanted to ask you something and I don't know where to contact you.
I thought that the project was abandoned because in the appstore is not seen that there have been updates for a long time but entering here to github I was surprised that they are constantly making changes but I do not understand why that is not sent to the appstore.

**Reply by StefanVDWeide** [reply]
Thank for letting me know! 

My contribution is just meant to show my appreciation for the hard work and I don't expect anything in return (features or whatever). I've used Swiftfin for about a year now and it has always worked great for me! 

Thanks again for the work! :)

**Comment by JPKribs** [comment]
I'm going to close this now this is part of our documentation in our README. The link for project donations can be found here:

https://jellyfin.org/docs/general/contributing/direct-donations/
